rest/method_post.go

```diff
@@ -47,6 +48,14 @@ func listPost(ctx context.Context, r *http.Request, route *RouteMatch) (status i
 	}
 	// See https://www.subbu.org/blog/2008/10/location-vs-content-location
 	headers = http.Header{}
-	headers.Set("Content-Location", fmt.Sprintf("%s/%s", r.URL.Path, item.ID))
+	itemID := item.ID
+	if f := rsrc.Validator().GetField("id"); f != nil {
+		if s, serialize := f.Validator.(schema.FieldSerializer); serialize {
```

Comment:
I think the following might be more idiomatic:

`s, ok := f..Validator.(schema.FieldSerializer); ok`

However, this is also OK.